mesh/pmesh.cpp

```diff
@@ -5568,6 +5568,97 @@ void ParMesh::PrintSharedEntities(const char *fname_prefix) const
    }
 }
 
+void ParMesh::GetGlobalVertexIndices(Array<HYPRE_Int> &gi)
+{
+   H1_FECollection fec(1, Dim); // Order 1, mesh dimension (not spatial dimension).
+   ParFiniteElementSpace fespace(this, &fec);
```

Comment:
I think all these functions should be `const`. It might be worth doing a `cast`, as below, to have the correct interface.
```suggestion
   ParMesh *pm = const_cast<ParMesh *>(this);
   ParFiniteElementSpace fespace(pm, &fec);
```